mail/gophish/files/gophish.init

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2006-2011 OpenWrt.org
+
+START=50
+STOP=50
+
+USE_PROCD=1
+PROG=/usr/sbin/gophish
+
+start_service() {
+	user_exists gophish || user_add gophish
+
+	[ -f /usr/libexec/gophish/gophish.db ] || {
+		touch /usr/libexec/gophish/gophish.db
+		chown gophish /usr/libexec/gophish/gophish.db
+		chmod 0640 /usr/libexec/gophish/gophish.db
+		chown gophish /usr/libexec/gophish/
+		chmod 750 /usr/libexec/gophish/
+	}
+
+	procd_open_instance
+	procd_set_param stdout 1
+	procd_set_param stderr 1
+	procd_set_param user gophish
+	procd_set_param env GOPHISH_VERSION="0.12.1"
```

Comment:
Why? I dont think this is correct.

Reply:
Do you mean the environment variable? The application requires this. Normally it is set by a file named VERSION in the application's working directory. I patched Gophish to support an environment variable instead to better suit procd.

Reply:
Ping @Jefferyto about this.